lesspipe.sh.in

```diff
@@ -669,7 +669,7 @@ isfinal() {
       msg "append :. or :<filetype> to activate syntax highlighting"
     else
       lang=${3#$sep}
-      lang="-l ${lang#.}"
```

Comment:
Could you make the same change to lesspipe.sh?

Reply:
Yes did it.

Thanks for reply.